### PR TITLE
docs(README.md): fix typo

### PR DESCRIPTION
**Pull request by char0n**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
 node_modules
+
+# Jetbrains metafiles
+.idea
```
Comment (kylef) [inline .gitignore]: This slipped in change doesn't belong in this PR or commit. Can you please remove.

**Review by kylef — COMMENTED** [review]
Thanks for PR, good to go once the one comment has been addressed.

**Comment by char0n** [comment]
.gitignore has been addressed

**Review by kylef — APPROVED** [review]
